notebooks/Annotation and Continuous Learning Overview.ipynb

```diff
@@ -699,21 +697,20 @@
     }
    ],
    "source": [
-    "# Pick a job id from the last cell output.\n",
-    "download_job_id = \"<job ID you want to download>\"\n",
-    "endpoint = f\"{base_url}/experiments/{experiment_id}/jobs/{download_job_id}:download\"\n",
-    "response = requests.get(endpoint, data=json.dumps({\"export_type\": \"monai_bundle\"}), headers=headers)\n",
-    "if response.status_code == 200:\n",
+    "# Pick a job id from the last cell output. For example, choose train_id\n",
+    "if \"train_id\" in locals():\n",
```

Comment:
Hi @mingxin-zheng , for the CL notebook, I added this logic to ensure the notebook runs fine even if there is no `train_id` (mainly used for CI test). Do you think this change is fine?